tests/Configurator/DockerComposeConfiguratorTest.php

```diff
@@ -114,7 +115,7 @@ class DockerComposeConfiguratorTest extends TestCase
     /** @var DockerComposeConfigurator */
     private $configurator;
 
-    /** @var Package */
+    /** @var PackageInterface */
```

Comment:
Trying here to fix: https://github.com/symfony/flex/runs/5509940830?check_suite_focus=true

But still errors this way: https://github.com/symfony/flex/runs/5510160271?check_suite_focus=true

Reply:
I think the right fix is to make it use a RootPackage instead of a Package for this property (but also for the actual value, not just for the comment)

Reply:
Thx that seems to fix the tests 👍 Had todo the same for LockArrayRepository in FlexTest.

Reply:
For backwards compatibility to composer 1.0.2 there a class_exists had to be added.